tla/consensus/Traceccfraft.tla

```diff
@@ -58,13 +58,13 @@ IsAppendEntriesResponse(msg, dst, src, logline) ==
 ASSUME TLCGet("config").mode = "bfs"
 
 JsonFile ==
-    IF "JSON" \in DOMAIN IOEnv THEN IOEnv.JSON ELSE "traces/election.ndjson"
+    IF "JSON" \in DOMAIN IOEnv THEN IOEnv.JSON ELSE "../../build/startup.ndjson"
 
 JsonLog ==
     \* Deserialize the System log as a sequence of records from the log file.
-     \* Run TLC with (assuming a suitable "tlc" shell alias):
-     \* $ JSON=../tests/raft_scenarios/4582.ndjson tlc -note Traceccfraft
-     \* Fall back to trace.ndjson if the JSON environment variable is not set.
+    \* Run TLC from under the tla/ directory with:
+    \* $ JSON=../build/startup.ndjson ./tlc.sh consensus/Traceccfraft.tla
```

Comment:
`../build/startup.ndjson` is inconsistent with `../../build/startup.ndjson` above.

Reply:
That's because one is run from the `tla/` directory, where the `tlc.sh` script `lives`, and the other is provided for the debugger, which seems to use the location of the spec file as the current working directory when it runs.

I agree it's not ideal, I'd prefer to remove the hardcoded path altogether and have a way to pass it to debugger, but I've not found a way to do that.

Reply:
@lemmy would TLC consider adding a way to pass CLI options to the code, maybe with a pass-through like [ansible's --extra-vars](https://www.redhat.com/sysadmin/extra-variables-ansible-playbook)?

Reply:
`IOUtils` has operators to read environment variables.  Perhaps, we should add similar operators to expose Java system properties.